xbmc/pvr/recordings/PVRRecording.cpp

```diff
@@ -97,7 +97,8 @@ CPVRRecording::CPVRRecording(const PVR_RECORDING& recording, unsigned int iClien
   if (strlen(recording.strFirstAired) > 0)
     m_firstAired.SetFromW3CDateTime(recording.strFirstAired);
   m_iFlags = recording.iFlags;
-  m_sizeInBytes = recording.sizeInBytes;
+  if (recording.sizeInBytes > 0)
```

Comment:
Strictly speaking >= 0. Theoretically one could change the init value of the recording size member to something <> 0, then your code breaks.

Reply:
Good point.